docs/content/Release-Notes/0-32.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: v0.32
+permalink: /releases/v0-32
+category: Release Notes
+releaseDate: 2023-03-02
+releaseLink: https://github.com/cube-js/cube.js/releases/tag/v0.32.0
+---
+
+Hello Cube Community, we're really excited to announce Cube v0.32 🎉
+
+This new release brings the GraphQL API into general availability, adds a number
+of quality-of-life improvements to Cube SQL and adds significant performance
+boosts to Cube Store.
+
+## Breaking Changes and Deprecations
+
+This release, we have **2** deprecations and **5** breaking changes:
+
+> Be sure to check the
+> [Deprecation document](https://github.com/cube-js/cube.js/blob/master/DEPRECATION.md)
+> for more details.
+
+### <--{"id" : "Breaking Changes and Deprecations"}--> Breaking Changes
+
+- Support for Node.js v12 and v15 has been dropped due to [them reaching
+  end-of-life][nodejs-eol]. We **strongly** recommend upgrading to Node.js v16.
+
+- Absolute imports for the following packages have been removed. We now provide
+  a public API from the package directly:
+
+  - `@cubejs-backend/query-orchestrator`
+  - `@cubejs-backend/schema-compiler`
+  - `@cubejs-backend/server-core`
+
+- Cube Store is now the default cache and query engine, replacing Redis. We've
```

Comment:
cache, queue and query engine) should we mention queue?

Reply:
It should have been queue, not query 😅